lib/msf/core/exploit/http/client.rb

```diff
@@ -448,16 +448,18 @@ def reconfig_redirect_opts!(res, opts)
 
       opts['ssl'] = ssl
```

Comment:
There's another one up here right?

Reply:
I think this line is basically a no-op because it will never differ from the previous request.